src/WingetCreateCore/Common/Extensions.cs

```diff
@@ -93,6 +93,16 @@ public static string ToEnumAttributeValue(this Enum enumVal)
             var attributes = memInfo[0].GetCustomAttributes(typeof(EnumMemberAttribute), false);
             EnumMemberAttribute attributeValue = (attributes.Length > 0) ? (EnumMemberAttribute)attributes[0] : null;
             return attributeValue?.Value ?? enumVal.ToString();
+        }
+
+        /// <summary>
+        /// Returns null if the target string is empty.
+        /// </summary>
+        /// <param name="s">Target string value.</param>
+        /// <returns>Null if the string is empty; otherwise the original string value.</returns>
+        public static string NullIfEmpty(this string s)

```

Comment:
Is this common practice?

Reply:
I believe it is, since it is commonly used with the null coalescing operator when handling empty strings